assets/src/block-validation/error/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { PanelBody, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AMP_VALIDITY_REST_FIELD_NAME,
+	VALIDATION_ERROR_ACK_ACCEPTED_STATUS,
+	VALIDATION_ERROR_ACK_REJECTED_STATUS,
+} from '../constants';
+import { NewTabIcon } from '../icon';
+import { ErrorPanelTitle } from './error-panel-title';
+import { ErrorContent } from './error-content';
+
+/**
+ * Component rendering an individual error. Parent component is a <ul>.
+ *
+ * @param {Object} props Component props.
+ * @param {string} props.clientId
+ * @param {number} props.status
+ * @param {number} props.term_id
+ */
+export function Error( { clientId, status, term_id: termId, ...props } ) {
```

Comment:
There is a naming collision between the rest `props` and the `props` in the jsdoc:

![image](https://user-images.githubusercontent.com/134745/102928817-458df080-444e-11eb-8b25-28bf727079f9.png)

This should fix it:

```suggestion
 * @param {Object} args Component props.
 * @param {string} args.clientId
 * @param {number} args.status
 * @param {number} args.term_id
 */
export function Error( { clientId, status, term_id: termId, ...props } ) {
```